src/App.css

```diff
@@ -1,38 +1,79 @@
-.App {
-  text-align: center;
+body{
+  background-image: url(background.png);
+  background-size: cover;
 }
+.todo-input{
+  width: 281px;
+  height: 40px;
```

Comment:
Typically for a more responsive design; `px` isn't the best option to set. you could better go with `rem` or using a percentage  `%`